opsdroid/parsers/tests/test_parser_format.py

```diff
@@ -0,0 +1,81 @@
+"""Test the opsdroid parse_format parser."""
+import asynctest.mock as amock
+
+from opsdroid.cli.start import configure_lang
+from opsdroid.core import OpsDroid
+from opsdroid.matchers import match_parse
+from opsdroid.events import Message
+from opsdroid.parsers.parseformat import parse_format
+
+
+configure_lang({})
+
+
+async def getMockSkill():
+    async def mockedskill(opsdroid, config, message):
+        pass
+
+    mockedskill.config = {}
+    return mockedskill
+
+
+async def test_parse_format_match_condition():
+    with OpsDroid() as opsdroid:
+
+        mock_skill = await getMockSkill()
+        opsdroid.skills.append(match_parse("Hello")(mock_skill))
+
+        mock_connector = amock.CoroutineMock()
+
+        message = Message("Hello world", "user", "default", mock_connector)
+        skills = await parse_format(opsdroid, opsdroid.skills, message)
+        assert len(skills) == 0
+
+        message = Message("Hello", "user", "default", mock_connector)
+        skills = await parse_format(opsdroid, opsdroid.skills, message)
+        assert mock_skill == skills[0]["skill"]
+        assert skills[0]["message"] is message
+        """Test that the original object has had a new attribute added."""
```

Comment:
I think this should be changed to just a comment since we don't need multiple lines.
```suggestion
        # Test that the original object has had a new attribute added.
```